packages/create-cloudflare/src/frameworks/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "frameworks_clis_info",
+	"dependencies": {
+		"@angular/cli": "^0.4.5",
+		"create-astro": "3.1.5",
+		"create-docusaurus": "2.4.1",
+		"create-hono": "0.2.6",
+		"create-next-app": "13.4.2",
+		"create-qwik": "1.2.7",
+		"create-react-app": "5.0.1",
+		"create-remix": "1.16.0",
+		"create-solid": "0.2.26",
+		"create-svelte": "4.2.0",
+		"create-vue": "3.6.4",
+		"gatsby": "5.10.0",
+		"nuxi": "3.4.2"
+	},
+	"//1": "this package.json is only used to keep track of the frameworks cli dependencies",
+	"//2": "so that we can use dependabot to update these dependencies automatically",
+	"//3": "additionally it also contains a map that maps frameworks to their respective clis",
```

Comment:
ideally these should be at the top, but this:
https://github.com/cloudflare/workers-sdk/blob/478bed3b80ea353a5be4bd7056b92460a3cf4cd5/package.json#L45

causes prettier to reorder the package.json fields 😓 

(I don't think that the order is configurable given [this](https://github.com/keithamus/sort-package-json#i-would-like-this-tool-to-be-configurable-with-a-config-file-or-command-line-arguments) (from the package used under the hook by the prettier plugin))

So I think we can either leave the fields like this or add this file to the prettierignore 😕

Reply:
moved to the description field